drivers/wifi/nrf700x/Kconfig

```diff
@@ -401,3 +401,14 @@ config NRF_WIFI_BEAMFORMING
 config WIFI_NRF700X_SCAN_TIMEOUT_S
 	int "Scan timeout in seconds"
 	default 30
+
+config WIFI_NRF700X_SKIP_LOCAL_MAC
+	bool "Suppresses local MAC and reserved IANA Unicast MAC addresses in scan results"
+	help
+	  Local MAC addresses (The U/L bit is set to 1) are not presumed
+	  globally unique across all IEEE 802 networks. The locally
+	  administered bits of local MAC addresses are arbitrarily assignable
+	  under the condition that local MAC addresses are unique within a LAN.
+	  For operations such as geolocating devices based on MAC address, universal
+	  uniqueness being necessary, may require ignoring MAC addresses whose
+	  U/L bit is set.
```

Comment:
That would do :)
But what I meant is a simple reason why someone want to suppress them. E.g. memory/bandwidth/power/etc. savings...